GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1567 @@
+                    try
+                    {
+                        await ((IUser)signedInUser).ThumbnailPhoto.UploadAsync(fileStream, "application/image");
+                        Console.WriteLine("\nUser {0} was updated with a thumbnailphoto", signedInUser.DisplayName);
+                    }
+                    catch (Exception e)
+                    {
+                        Program.WriteError("\nError Updating the user photo {0} {1}", e.Message,
+                            e.InnerException != null ? e.InnerException.Message : "");
+                    }
+                }
+            }
+
+            #endregion
+
+            #endregion
+        }
+
+        private static async Task PeoplePickerExample(IActiveDirectoryClient client, string searchString)
+        {
+            #region People Picker example
+
+            //*********************************************************************
+            // People picker
+            // Search for a user using text string "Us" match against userPrincipalName, displayName, giveName, surname
+            // Requires minimum of User.ReadBasic.All.
+            //*********************************************************************
+
+            List<IUser> usersList = null;
+            IPagedCollection<IUser> searchResults = null;
+            try
+            {
+                IUserCollection userCollection = client.Users;
+                searchResults = await userCollection.Where(user =>
+                    user.UserPrincipalName.StartsWith(searchString) ||
+                    user.DisplayName.StartsWith(searchString) ||
+                    user.GivenName.StartsWith(searchString) ||
+                    user.Surname.StartsWith(searchString)).Take(10).ExecuteAsync();
+                usersList = searchResults.CurrentPage.ToList();
+            }
+            catch (Exception e)
+            {
+                Program.WriteError("\nError getting User {0} {1}", e.Message,
+                    e.InnerException != null ? e.InnerException.Message : "");
+            }
+
+            if (usersList != null && usersList.Count > 0)
+            {
+                do
+                {
+                    int j = 1;
+                    usersList = searchResults.CurrentPage.ToList();
+                    foreach (IUser user in usersList)
+                    {
+                        Console.WriteLine("User {2} DisplayName: {0}  UPN: {1}",
+                            user.DisplayName, user.UserPrincipalName, j);
+                        j++;
+                    }
+                    searchResults = await searchResults.GetNextPageAsync();
+                } while (searchResults != null);
+            }
+            else
+            {
+                Console.WriteLine("User not found");
+            }
+
+            #endregion
+        }
+
+        private static async Task<User> SearchForUserByUpn(
+            IActiveDirectoryClient client,
+            string searchString,
```

Comment:
>searchString [](start = 19, length = 12)

This input param value is not used anywhere. It gets assigned a new value as a first line in the code.